pkg/status/status_process_agent.go

```diff
@@ -16,10 +16,12 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
+// httpClients should be reused instead of created as needed. They keep cached TCP connections
+// that may leak otherwise
+var httpClient = apiutil.GetClient(false)
```

Comment:
Rather than a global variable, could this be a `sync.Once` type of scenario?

Reply:
I had the same thought, but GetClient just allocates an empty-ish struct, without actually calling anything (e.g., creating a socket), so I think this is OK.

Reply:
I updated it to use `sync.Once` and only initialize `httpClient` when `initHTTPClient` is used:
https://github.com/DataDog/datadog-agent/pull/11083/commits/95bb4b2a42005e0b4bcf19250dfc7504aa880979

Please let me know what do you think abou it!